### ISSUE TITLE
Help Manual - Dashboard: "language preferences" links to "Editing your profile" page

### ISSUE THREAD

**Issue by XiuJuanXia** [mmiron-HC/WST-RSAMS#71]
URL: https://hcdev.openplus.ca/en/documentation/dashboard-all-roles
User Agent: Chrome
Language: Both

Issue Description: In the fourth bullet, "language preferences" links to "Editing your profile" page. Related information is not presented to users right away. Users will have to move down to view the related information. Same issue applies to the French page. Also the link on the French page directs users to English page.
![image](https://user-images.githubusercontent.com/88776404/129194263-bc305a91-6ae6-44c2-9089-c8d05be47385.png)

Steps to reproduce: Moving focus to the "language preferences" link and select it.

Expected results: Adding an anchor to the destination page so that users can view the related information right away. In the French page, link should be updated to the proper French URL.

**Comment by echouin** [comment]
Currently, we are unable to add Anchors via the Drupal WYSIWYG.
I have modified the direct link to go to the "editing your profile" section (which is more relevant to the bullet). This resolves the issue.

EN:
![image](https://user-images.githubusercontent.com/88099454/148392438-3601498e-974c-4085-b060-aefa3860c1d2.png)
FR:
![image](https://user-images.githubusercontent.com/88099454/148392500-5584629c-9a8d-4e2a-a1a9-031d7a63769e.png)

**Comment by hydegeoff** [comment]
Not fixed, because the link's URL has the old site hard-coded into it, and it is probably causing Broken in PROD.  links should be using relative URLs instead of using the full URL.  See the attached screenshot:  The first two "user role" links probably have broken URLs: 
![Old site URL is hard coded into the links](https://user-images.githubusercontent.com/88147013/160000723-f12fc86b-66f5-4324-9e2d-7dcd3df061a5.jpg)

**Comment by echouin** [comment]
@hydegeoff thanks for catching. All links fixed on PROD.

**Comment by hydegeoff** [comment]
@echouin, can you please provide a screenshots with the updated URL, so I can close the issue.  (in FR and EN)

**Comment by echouin** [comment]
Hi @hydegeoff the URLs were updated to: https://rsams.prod.cloud.openplus.ca/en/documentation/user-roles

![image](https://user-images.githubusercontent.com/88099454/160472116-81a66adc-591d-426b-baf0-740d385c74c6.png)

**Comment by hydegeoff** [comment]
@echouin, May I please have a screenshot of the page and the associated source code so I can see URL has been changed.   (like the screenshot I took 4 days ago)

**Comment by echouin** [comment]
@hydegeoff 
![image](https://user-images.githubusercontent.com/88099454/160476787-7e849364-1153-4ebd-b466-22f1fbc2c265.png)

**Comment by mmiron-HC** [comment]
@echouin This isn't quite what we need. For each link, we need to see the code behind it, the <a> with the href. like this 
![image](https://user-images.githubusercontent.com/65728700/160616465-bba54433-9bed-4f9e-9260-2c92bb03f5e7.png)

**Comment by echouin** [comment]
@mmiron-HC can you please tell me how to see this code?

I had googled and hit F12?

OR, please check the page found here on PROD: https://rsams.prod.cloud.openplus.ca/en/documentation/user-roles

**Comment by davidgotlieb** [comment]
@echouin Hey Eric, if you're in Chrome go to the three dots at the top right > More tools > Developer tools

**Comment by echouin** [comment]
![image](https://user-images.githubusercontent.com/88099454/160624189-a3119dd1-164f-4419-92f2-c2eac0486248.png)

**Comment by hydegeoff** [comment]
Based on the screenshot in PROD, this issue can be closed.
